modules/rgbd/include/opencv2/rgbd/large_kinfu.hpp

```diff
@@ -17,6 +17,68 @@ namespace cv
 {
 namespace large_kinfu
 {
+struct CV_EXPORTS_W VolumeParams
+{
+    /** @brief Kind of Volume
+            Values can be TSDF (single volume) or HASHTSDF (hashtable of volume units)
+    */
+    CV_PROP_RW VolumeType kind = VolumeType::TSDF;
+
+    /** @brief Resolution of voxel space
+        Number of voxels in each dimension.
+        Applicable only for TSDF Volume.
+        HashTSDF volume only supports equal resolution in all three dimensions
+    */
+    CV_PROP_RW int resolutionX = 128;
+    CV_PROP_RW int resolutionY = 128;
+    CV_PROP_RW int resolutionZ = 128;
```

Comment:
Each member should be documented.

Why we can't use `Vec3i`?

Reply:
The true reason to use 3 ints instead of `Vec3i` is that `Vec3i` does not work well with bindings generator